pkg/library/kuadrant/apimachinery_status_conditions.go

```diff
@@ -78,3 +81,27 @@ func EnforcedCondition(policy Policy, err PolicyError, fully bool) *metav1.Condi
 
 	return cond
 }
+
+func HealthyCondition(policy Policy, err PolicyError) *metav1.Condition {
```

Comment:
not 100% sure if this should be here or not @mikenairn ?

Reply:
Are you expecting this to ever be used by anything other than DNSPolicy? Seems unlikely, I'd probably move it all into the dns_workflow

Reply:
It is strange to have a condition defined in the workflow and not alongside other conditions :thinking:  don't have a strong opinion on this, however - moved